main.go

```diff
@@ -48,17 +53,27 @@ func init() {
 	utilruntime.Must(clientgoscheme.AddToScheme(scheme))
 
 	utilruntime.Must(crunchybridgev1alpha1.AddToScheme(scheme))
-	utilruntime.Must(dbaasredhatcomv1alpha1.AddToScheme(scheme))
-	utilruntime.Must(dbaasoperator.AddToScheme(scheme))
 	//+kubebuilder:scaffold:scheme
 }
 
+type mainConfig struct {
+	apiURL    string
+	clientset *kubernetes.Clientset
+}
+
+var dbaasInit func(ctrl.Manager, mainConfig)
+
 func main() {
-	var metricsAddr string
+	// Variables from boilerplate
+	var metricsAddr, probeAddr string
 	var enableLeaderElection bool
-	var probeAddr string
+
 	var crunchybridgeAPIURL string
-	flag.StringVar(&crunchybridgeAPIURL, "crunchybridgeapi-url", "https://api.crunchybridge.com", "the Crunchy bridge API URL (no slash in the end).")
+	// Namespace and Name for APIKey secret default values
+	credNamespace := "default"
+	credName := "crunchybridge_api_key"
```

Comment:
same we can put into CR?

Reply:
These variables are not used when DBaaS support is enabled, they are only for 'standalone operator' use.